integration/kubernetes/k8s-block-volume.bats

```diff
@@ -33,6 +37,8 @@ setup() {
 	sed -i "s|HOSTNAME|$(hostname)|" "$tmp_pv_yaml"
 	sed -i "s|CAPACITY|${vol_capacity}|" "$tmp_pv_yaml"
 	kubectl create -f "$tmp_pv_yaml"
+	cmd="kubectl get pv/${volume_name} | grep Available"
+	waitForProcess "$wait_time" "$sleep_time" "$cmd"
```

Comment:
I was hoping that we could make use of `kubectl wait` here, but looks like it does not work for persistent volume resources.

Reply:
@amshinde same here. I tried to use `kubectl wait` but found no way to check the status is available. :(

Reply:
same concern. This is at least better than what we had...